packages/kotti-popover/src/Popover.vue

```diff
@@ -20,6 +20,16 @@ export default {
 		placement: { type: String, default: 'bottom' },
 		size: { type: String, default: null },
 		content: { type: String, default: '' },
+		showPopperProp: {
```

Comment:
Hi @Juli0GT, great addition and nice work. Regarding the naming, I would consider to rename the prop to `forceShowPopover`. since force is a bit clearer that it indicates an override of normal behaviour and popover is more descriptive than the library name popper.

Reply:
Yeah good point @mvangent I'll rename the prop.